rel/overlay/etc/default.ini

```diff
@@ -187,6 +188,14 @@ credentials = false
 ; List of accepted methods
 ; methods =
 
+[x_frame_options]
+; Settings same-origin will return X-Frame-Options: SAMEORIGIN.
+; If same origin is set, it will ignore the hosts setting
+; same_origin = true
+; Settings hosts will return X-Frame-Options: ALLOW-FROM https://example.com/
+; List of hosts separated by a comma. * means accept all
+; hosts =
```

Comment:
I think its the same as for cors or anything else. The typo's will give incorrect results and mean that they might have a situation where a page can be or cannot be embedded